Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM rust:1-jessie as builder
+
+WORKDIR /srv/cosmogony
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y make libgeos-c1 libgeos-dev libssl-dev git && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY . ./
+
+RUN cargo build --release
+
+FROM debian:jessie-slim
+
+WORKDIR /srv
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y libcurl3 libgeos-c1 libgeos-dev libssl-dev && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY --from=builder /srv/cosmogony/target/release/cosmogony /usr/bin/cosmogony
+
+ENTRYPOINT cosmogony
```

Comment:
For some reason, I think that an array `["cosmogony"]` is required in order to pass the args properly.  
Here `docker run cosmogony -h` returns an error, instead of the expected help message (easy to miss as they are similar).

Reply:
:man_facepalming: